package.json

```diff
@@ -66,6 +66,8 @@
     ]
   },
   "resolutions": {
+    "**/@react-native-community/**/fast-xml-parser": "^4.4.1",
+    "**/@aws-*/**/fast-xml-parser": "^4.4.1",
```

Comment:
oooh, I didn't know you could do this

Reply:
It's pretty useful, def saved on duplication here